lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -0,0 +1,302 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone', 'js/vendor/backbone-super',
+    ], function (gettext, $, _, Backbone) {
+
+        var messageRevertDelay = 4000;
+        var AccountSettingsFieldViews = {};
+
+        AccountSettingsFieldViews.FieldView = Backbone.View.extend({
+
+            className: function () {
+                return "account-settings-field " + "account-settings-field-" + this.options.valueAttribute;
+            },
+
+            tagName: 'div',
+
+            indicators: {
+                'error': '<i class="fa fa-exclamation-triangle message-error"></i>',
+                'invalidInput': '<i class="fa fa-exclamation-triangle message-invalid-input"></i>',
+                'inProgress': '<i class="fa fa-spinner message-in-progress"></i>',
+                'success': '<i class="fa fa-check message-success"></i>',
+            },
+
+            messages: {
+                'error': gettext('An error occurred, please try again.'),
```

Comment:
Yeah, I scared @catong away. RAWR!! :-) 

What's the context of this error message? Could it say, for example, "An error occurred. Try saving your settings again."?